classes/event/element_updated.php

```diff
@@ -87,6 +87,26 @@ public static function create_from_element(\mod_customcert\element $element) : e
         return self::create($data);
     }
 
+    /**
+     * Create instance of event for the specified element.
+     *
+     * @param int $elementid ID of the element.
+     * @param int $templateid ID of template containing the above element.
+     * @return element_updated
+     */
+    public static function create_from_ids(int $elementid, int $templateid): element_updated {
+        global $DB;
+
+        $template = $DB->get_record('customcert_templates', ['id' => $templateid]);
```

Comment:
To avoid having do this query why not pass in the template object that is created above? It will save you a query.

Reply:
Thanks for the feedback, revised patch with the suggested improvement.

Reply:
Thanks.